ONSdigital/census-rm-performance-tests
# Update sample files and add required field to receipting test

# Motivation and Context

The sample frame has been updated so now all the sample files need to be updated. 
# What has changed



Samples have been regenerated using the sample loader changes.
The receipting test was also failing due to it requiring "unrecipted:" in the json event so this has been added.

# How to test?



Check the sample files, try running some tests.
Give the large sample files a quick review. If possible, download these (should only take a few minutes) and give them a quick eyeball. These new sample files are located in the performance env, in the bucket called `census-rm-performance-sample-files-tmp` and the two files to look at are called:
`30_million_household_sample_to_be_reviewed.csv`
`3_5_million_sample_to_be_reviewed.csv`
# Links



https://trello.com/c/xTAp838U/777-update-performance-tests-following-sample-frame-change-3
# Screenshots (if appropriate):
